src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1346,11 +1344,20 @@ namespace winrt::TerminalApp::implementation
             }
         }
 
+        _RemoveTab(tab);
+    }
+
+    // Method Description:
+    // - Removes the tab (both TerminalControl and XAML)
+    // Arguments:
+    // - tab: the tab to remove
+    void TerminalPage::_RemoveTab(winrt::TerminalApp::TabBase tab)
```

Comment:
Now, because this one is **not** a coroutine we should capture the TabBase by `const &`.

We could not do it in the previous function because it was a coroutine and the behavior of references across coroutines is "it will crash, yo"

Reply:
:smile: yeah absolutely! I will fix that!